librocksdb_sys/crocksdb/c.cc

```diff
@@ -1768,7 +1768,7 @@ void crocksdb_options_set_compression(crocksdb_options_t* opt, int t) {
   opt->rep.compression = static_cast<CompressionType>(t);
 }
 
-int crocksdb_options_get_compression(crocksdb_options_t* opt) {
+int crocksdb_options_get_compression(crocksdb_options_t *opt) {
```

Comment:
I don't think so. We should use *librocksdb_sys/crocksdb/format-diff.sh* to format c/c++ codes, which is adapted from [rocksdb](https://github.com/facebook/rocksdb/blob/master/build_tools/format-diff.sh).